src/main/webapp/style.css

```diff
@@ -0,0 +1,90 @@
+body{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.navbar{
+  width: 100%;
+  max-width: 1400px; 
+  flex-flow: row nowrap;
+}
+
+.form-line{
+    max-width: 60vw;
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    flex-grow: 1;
+}
+
+.wrapper{
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  background-color: #A9A9A9;
+  width: 100%;
+}
+
+#header{
```

Comment:
This would then rename to `.header`, or make it more unique by changing it to `.site-header`.